include/swift/Syntax/CMakeLists.txt

```diff
@@ -17,3 +17,7 @@ add_gyb_target(swift-syntax-generated-headers
     "${generated_include_sources}")
 set_property(TARGET swift-syntax-generated-headers
     PROPERTY FOLDER "Miscellaneous")
+
+if(TARGET clang)
+  add_dependencies(swift-syntax-generated-headers clang)
```

Comment:
Why the dependency on `clang` here?  Perhaps `gyb` should depend on `clang` if that needs it?

Reply:
Apparently, this target uses the just-built Clang, while everything else is built with my host compiler.

Reply:
How?  These are headers which are generated by gyb, it shouldn't be using clang.

Reply:
I don't know, for me it fixed a symptom. Maybe it's obsolete with `SWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER`? Will try.